File: Sem 4/Flutter/Lab 4/fibonaci.dart
import 'dart:io';

void fibonaci(int n) {
  int a = 0, b = 1, c = 0;
  for (int i = 0; i < n; i++) {
    print("$a");
    c = a + b;
    a = b;
    b = c;
  }
}

void main() {
  stdout.write("Enter num : ");
  int n = int.parse(stdin.readLineSync()!);
  fibonaci(n);
}

File: Sem 4/Daa/lab 1/fibonaci.c
#include <stdio.h>

int fibonacci(int n);

int main () {
    int n;
    printf("Enter num");
    scanf("%d", &n);

    printf("%d",fibonacci(n));
}

int fibonacci(int n) {
    if (n <= 1) {
        return n;
    } else {
        
        return fibonacci(n - 1) + fibonacci(n - 2);
    }
}